### DISCUSSION TITLE
Add ability to remove non-silence disturbances

### DISCUSSION THREAD

**Discussion by luca-ing** [WyattBlue/auto-editor#418] [2. Feature Ideas]
Hi WyattBlue,

I've been using auto-editor with great success, and am very thankful and happy you shared it with us.
I use it to generate cut lists to cut podcasts.

Apart from excising overly long silences, I also need to remove an excess of um, eh, ah, and similar utterances.
I've cobbled together something vaguely workable based on openai-whisper, but it's note pretty: for now processing media needs to be a two-step process: run auto-editor, apply its cut list render, then run my utterances detection on the rendered output and apply its cut list.

What I have to work with are timestamps for beginning and end of words.
I've spent yesterday staring at your code, trying to figure out where to splice in my work so that I might generate a cut list that combines both, but apparently my programming skills aren't up to the task.

I know you've declined feature requests in this direction in the past, but I would be grateful if you could give me some hints on where to look, and consider if you'd be interested in taking a PR eventually.

**Comment by WyattBlue** [comment]
@luca-ing The best thing you can do is send a working stand-alone script that just outputs to the terminal. Don't worry about edge-cases, just make an minimal viable product.

Don't try to integrate auto-editor. I can do that far faster than you can because I know the codebase like the back of my hand.
